docs/newbs_getting_started.md

```diff
@@ -166,6 +166,8 @@ For example, to build a firmware for a Clueboard 66% you would use:
 
     qmk compile -kb clueboard/66/rev3 -km default
 
+?> The keyboard option is the path relative to the keyboard directory, the above example would be found in `qmk_firmware/keyboards/clueboard/66/rev3`. If you're unsure you can view a full list of supported keyboards with `qmk list-keyboards`.
```

Comment:
Might be worth mentioning that it supports autocomplete/tab completion, too.  eg `qmk -kb moon(tab) will show "moon" and "moonlander" as options.   May not work on every system, though.